test/units/modules/network/fortimanager/test_fmgr_script.py

```diff
@@ -0,0 +1,50 @@
+# (c) 2016 Red Hat Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.modules.network.fortimanager import fmgr_script
+from units.modules.utils import set_module_args
+from .fortimanager_module import TestFortimanagerModule
```

Comment:
Imports need to be written so the test is gracefully skipped (`raise SkipTest`) if they are not present, see `test/units/modules/network/f5/test_bigip_monitor_snmp_dca.py` for an example